lib/interface.go

```diff
@@ -37,6 +39,22 @@ func SearchByID(repo Repository, id int) (data IDCarrier, err error) {
 	}
 }
 
+func SearchByIDPrefix(repo Repository, prefix string) (id int, err error) {
```

Comment:
I think that `CompleteItemIDByIDPrefix` is more appropriate for that function name.